check-log/lib/check-log.go

```diff
@@ -431,6 +415,39 @@ func (opts *logOpts) match(line string) (bool, []string) {
 	return true, matches
 }
 
+func parseFilePattern(directory string, filePattern string, caseInsensitive bool) ([]string, error) {
```

Comment:
This signature can be more simply: `parseFilePattern(directory, filePattern string, caseInsensitive bool)...`
https://play.golang.org/p/mh-vr-IkqhH